site/topics/searching/searching.rst

```diff
@@ -54,32 +54,77 @@ Searching (again)
 
 	<iframe width="560" height="315" src="https://www.youtube.com/embed/8unA_a5xcCU" frameborder="0" allowfullscreen></iframe>
 
+
+Binary Search
+=============
+
 .. admonition:: Activity
     :class: activity
 
-    James will now guess a number between 0 and 1023 in less than 10 guesses. 
-   
-.. admonition:: Activity++
-    :class: activity
+    I will guess a number between :math:`0 -- 1023` in :math:`10` or less guesses.
```

Comment:
May be more intuitive to search for a word in a dictionary. People use interpolation search (looking for "poop" you'd open the book to where you'd think P is, then correct based on your understanding of the distribution of first letters of words), binary search is basically that except you ignore any statistical properties of the underlying data -- close enough.

...are zoomers aware of dictionaries? maybe the number thing is fine

Reply:
Here's why I'll leave it:
- Interpolation search is how it is introduced in the subsequent course
- _Interpolation search -> binary search -> analysis_ is more dots to connect than _binary search -> analysis_
- They probably have played this higher/lower game before anyways
- And above all else, which cannot be stressed enough, like at least 99.98% of the reason, that change would require more than 0 effort.

Reply:
I wasn't suggesting going into interpolation search, rather that people intrinsically use that algorithm. The gap from interpolation to binary is trivial, so an example where people use interpolation search inherently may be a better example.

Numbers are still fine.